static/js/constants.js

```diff
@@ -12,6 +12,8 @@ export const DASHBOARD_FORMAT = 'MMM D, Y';
 export const DASHBOARD_MONTH_FORMAT = 'MM[/]YYYY';
 
 export const CP1252_REGEX = /^[\u0020-\u00FF]*$/;
+// greater-than, comma, quotation-mark
```

Comment:
can we use `closing angle bracket` instead of `greater-than`. Because this is not related to programming,
 it is a text.